_IBC/dungeon-migaloo.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$schema": "../ibc_data.schema.json",
+  "chain_1": {
+    "chain_name": "dungeon",
+    "client_id": "07-tendermint-10",
+    "connection_id": "connection-12"
+  },
+  "chain_2": {
+    "chain_name": "migaloo",
+    "client_id": "07-tendermint-139",
+    "connection_id": "connection-109"
+  },
+  "channels": [
+    {
+      "chain_1": {
+        "channel_id": "channel-6",
```

Comment:
Error: For chain: dungeon, channel_id: channel-6 is registered for both: migaloo and terra2.

Reply:
@JeremyParish69 Corrected to channel-7